salt/kube-controller-manager/init.sls

```diff
@@ -0,0 +1,23 @@
+include:
+  - repositories
```

Comment:
The comment above about moving repos and certs -> kube-common fixes this, but.. If this was left as is, and kube-controller-manager was deployed onto it's own server, the certs would be missing :)